### ISSUE TITLE
Additional zoom level

### ISSUE THREAD

**Issue by claudep** [rkaravia/Leaflet.TileLayer.Swiss#6]
I have a use case, where I need an additional non-native zoom for a map. For the zoom level, I can easily add a `maxZoom: 29` option, however, an additional resolution is also needed (`0.05`) so I have to change the Leaflet.TileLayer.Swiss.js file to include that resolution. Do you think it would be doable to allow some additional option for that, or maybe automatically add a resolution depending on maxZoom?

**Comment by rkaravia** [comment]
Yes, I think the way to go would be to allow providing your own array of resolutions, I will try to add this in the future.
